app/src/main/java/pl/szczodrzynski/edziennik/ui/modules/timetable/TimetableDayFragment.kt

```diff
@@ -102,14 +117,17 @@ class TimetableDayFragment : LazyFragment(), CoroutineScope {
                 val events = withContext(Dispatchers.Default) {
                     app.db.eventDao().getAllByDateNow(App.profileId, date)
                 }
-                processLessonList(lessons, events)
+                val attendances = withContext(Dispatchers.Default) {
```

Comment:
In new code we're trying to use just `attendance` (or `attendanceList`) as there's no plural form of this word.

Reply:
https://www.learnersdictionary.com/definition/attendance

Reply:
yes but it looks weird